server/config/config.go

```diff
@@ -12,15 +12,14 @@ import (
 	flagyaml "github.com/buildbuddy-io/buildbuddy/server/util/flagutil/yaml"
 )
 
-const pathFlagName = "config_file"
-
-var configPath = flag.String(pathFlagName, "/config.yaml", "The path to a buildbuddy config file")
-
-func init() {
-	// As this flag determines the YAML file we read the config from, it can't
-	// meaningfully be specified in the YAML config file.
-	flagyaml.IgnoreFlagForYAML(pathFlagName)
-}
+// As this flag determines the YAML file we read the config from, it can't
+// meaningfully be specified in the YAML config file.
+var configPath = flagutil.New(
```

Comment:
Is there a need to wrap after every argument? It's inconsistent with how we typically define our flags.

Reply:
There is not, I'll swap it back. The line was just long, so I wrapped it.